### DISCUSSION TITLE
Projects: new workflow - Item/Issue assigned to Pull Request

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#73222] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

I want to have a new workflow to update an item/issue in a project, specifically I want to update the status to "In Progress", whenever an issue is assigned/linked to a Pull Request.
![image](https://github.com/community/community/assets/20150243/5293d176-024b-4cf3-8be7-10cdd007ab0c)

At the moment, when someone opens a Pull Requests and links an Issue to it, the issue is not updated and still in the backlog. A user has to manually update the issue and set/move it to "In Progress".

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ignacio-borrajo** [comment]
No luck about this? I am in the same situation and it would be very useful to have this option available

**Comment by cschreib-ibex** [comment]
I was also looking for this feature. This is something competitors already do (e.g. ZenHub) and I was trying to replicate in GitHub.
